widget/graphic.go

```diff
@@ -31,6 +31,11 @@ type Graphic struct {
 type GraphicImage struct {
 	Idle     *ebiten.Image
 	Disabled *ebiten.Image
+
```

Comment:
Can you please either enable Graphic to use these two images, or create a new ticket to do so?